src/dataDisplay/chip/chip.jsx

```diff
@@ -0,0 +1,228 @@
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+import MuiChip from '@material-ui/core/Chip';
+import React from 'react';
+import {
+    BEM_CHIP,
+    UI_CLASS_NAME,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+import Icon from '../icon';
+
+const propTypes = {
+    /**
+     * Additional classes.
+     */
+    className: PropTypes.string,
+    /**
+     * Override or extend the styles applied to Chip.
+     */
+    classes: PropTypes.shape({
+        deleteIcon: PropTypes.string,
+        disabled: PropTypes.string,
+        label: PropTypes.string,
+        root: PropTypes.string,
+    }).isRequired,
+    /**
+     * Override the default delete icon element. Shown only if `onDelete` is set.
+     */
+    deleteIcon: PropTypes.element,
+    /**
+     * If `true`, the chip should be displayed in a disabled state.
+     */
+    disabled: PropTypes.bool,
```

Comment:
Should this be `disable` for consistency with other React CM UI components?  Does MUI use `disabled`?

Reply:
MUI uses disabled. I am trying to get everything in sync as best as I can with them.

Reply:
Is that gonna cause us to rethink the rest of React CM UI (where we previously standardized on `disable`)?  Just wondering ... not necessarily arguing one way or the other. 😁

Reply:
yeah, probably. I think we'll go back to old `disabled` eventually.